packages/block-editor/src/components/list-view/block.js

```diff
@@ -245,17 +237,13 @@ function ListViewBlock( {
 			id={ `list-view-block-${ clientId }` }
 			data-block={ clientId }
 			isExpanded={ canExpand ? isExpanded : undefined }
-			aria-selected={ !! isSelected || forceSelectionContentLock }
 			ref={ rowRef }
 		>
 			<TreeGridCell
 				className="block-editor-list-view-block__contents-cell"
 				colSpan={ colSpan }
 				ref={ cellRef }
-				aria-label={ blockAriaLabel }
 				aria-selected={ !! isSelected || forceSelectionContentLock }
-				aria-expanded={ canExpand ? isExpanded : undefined }
-				aria-describedby={ descriptionId }
```

Comment:
Just to make sure that I'm following along, the main change here is that the attributes are being set on the block contents link instead of the overall cell? I'm not too sure of the implications, but this sounds good to me so far.

Reply:
Yes.